gnovm/Makefile

```diff
@@ -45,6 +45,10 @@ build:
 install:
 	go install $(GOBUILD_FLAGS) ./cmd/gno
 
+.PHONY: install.debug
+install:
+	go install $(GOBUILD_FLAGS) -tags debug ./cmd/gno
```

Comment:
this is a duplicate; please remove or update the previous one

Reply:
This is with the `-tags debug` flag; I don't think it's a duplicate?